src/diffusers/models/attention_processor.py

```diff
@@ -217,6 +223,17 @@ def __init__(
         if self.context_pre_only is not None and not self.context_pre_only:
             self.to_add_out = nn.Linear(self.inner_dim, self.out_dim, bias=out_bias)
 
+        if qk_norm is not None and added_kv_proj_dim is not None:
+            if qk_norm == "layer_norm":
+                self.norm_added_q = nn.LayerNorm(dim_head, eps=eps)
+                self.norm_added_k = nn.LayerNorm(dim_head, eps=eps)
```

Comment:
```suggestion
```
let's add this later when we have a model need this configuration, with joint attention + the regular "layer_norm" qk norm